test/test_va_api_config_attribs.cpp

```diff
@@ -39,40 +40,106 @@ class VAAPIConfigAttribs
     {
         doTerminate();
     }
+
+    void validateConfigAttributes(const ConfigAttributes& actual,
+        const ConfigAttributes& supported) const
+    {
+        const size_t size(actual.size());
+        ASSERT_EQ(size, supported.size());
+
+        // Require that actual and supported are in the same order, by type
+        for (size_t i(0); i < size; ++i) {
+            const VAConfigAttrib& aAttrib = actual[i];
+            const VAConfigAttrib& sAttrib = supported[i];
+
+            ASSERT_EQ(aAttrib.type, sAttrib.type);
+
+            switch (aAttrib.type) {
```

Comment:
Just to make a note: not all aAttrib.type are covered under this switch case. There are other bit field cases which needs to be covered. ie. VAConfigAttribFEIFunctionType and more. This should be in #ToDo list.